include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -4921,6 +4921,44 @@
       "capabilities" : [ "ShaderClockKHR" ],
       "version" : "None"
     },
+    {
+      "opname" : "OpFinalizeNodePayloadsAMDX",
+      "class"  : "Reserved",
+      "opcode" : 5075,
+      "operands" : [
+        { "kind" : "IdRef", "name": "'Payload Array'" }
+      ],
+      "capabilities" : [ "ShaderEnqueueAMDX" ],
+      "extensions" : [ "SPV_AMDX_shader_enqueue" ],
```

Comment:
Only capabilities introduced by the extension should be enabled by extensions. Instructions and other enumerants should only be enabled by capability.